templates/download/core/raspberry-pi-compute-module-3.html

```diff
@@ -0,0 +1,145 @@
+
+<section class="p-strip is-deep is-bordered">
+  <div class="row">
+    <div class="col-12">
+      <h2>Installation instructions</h2>
+      <ol class="p-stepped-list--detailed">
+        {% include "./_setup-ubuntu-sso.html" with number=1 %}
+        <li class="p-list-step__item">
+          <h3 class="p-list-step__title">
+            <span class="p-list-step__bullet">2</span>
+            Download Ubuntu Core
+          </h3>
+          <div class="p-list-step__content">
+            <p>Get the correct Ubuntu Core image for your board:</p>
+            <ul class="u-no-margin--left">
+              <li><a class="p-link--external" href="http://cdimage.ubuntu.com/ubuntu-core/16/stable/current/ubuntu-core-16-cm3.img.xz">Ubuntu Core 16 image for Compute Module 3</a></li>
+              <li>You can verify the integrity of the files using the <a class="p-link--external" href="http://cdimage.ubuntu.com/ubuntu-core/16/stable/current/SHA256SUMS">SHA256SUM</a> and <a class="p-link--external" href="http://cdimage.ubuntu.com/ubuntu-core/16/stable/current/SHA256SUMS.gpg">SHA256SUM.gpg</a> files.</li>
+            </ul>
+          </div>
+        </li>
+        <li class="p-list-step__item">
+          <h3 class="p-list-step__title">
+            <span class="p-list-step__bullet">3</span>
+            Setup USBboot on your host system
+          </h3>
+          <div class="p-list-step__content">
+            <p>On the host system: Ubuntu Desktop 16.04 or above:</p>
+            <ol>
+              <li>
+                <p>In a terminal, download the USBboot tool you will use to setup the board, and install its build dependencies:</p>
+                <pre><code>git clone --depth=1 https://github.com/raspberrypi/usbboot.git
+sudo apt install libusb-1.0-0-dev</code></pre>
+              </li>
+              <li>
+                <p>Then <code>cd</code into the USBboot directory, build with <code>make</code> and start the resulting binary as root:</p>
+                <pre><code>cd usbboot
+make
+sudo ./rpiboot</code></pre>
+              </li>
+              <li>Once started, it will wait for the Compute Module to be attached to the machine.</li>
+            </ol>
+          </div>
+        </li>
+        <li class="p-list-step__item">
+          <h3 class="p-list-step__title">
+            <span class="p-list-step__bullet">4</span>
+            Setup the Compute Module IO board
+          </h3>
+          <div class="p-list-step__content">
+            <p>On the Compute Module IO board:</p>
+            <ol>
+              <li>Position the Compute Module on the IO board.</li>
+              <li>Attach the USB hub, RJ45 adaptor, keyboard and monitor (HDMI) to the board.</li>
+              <li>
+                <p>Ensure the <code>J4</code> switch (<code>USB SLAVE BOOT ENABLE</code>) on the IO board is in the EN position.</p>
+                <img class="p-image--bordered" href="{{ ASSET_SERVER_URL }}55329e6f-CM3_J4.JPG?w=300" alt="J4">
```

Comment:
not sure why, but these images are loading

![image](https://user-images.githubusercontent.com/441217/36230349-0d63aca6-11d2-11e8-86c2-fc45ca7dc669.png)